pushy/src/main/java/com/turo/pushy/apns/server/MockApnsServerHandler.java

```diff
@@ -294,19 +304,21 @@ private void handleEndOfStream(final ChannelHandlerContext context, final Http2S
     public void write(final ChannelHandlerContext context, final Object message, final ChannelPromise writePromise) {
         if (message instanceof AcceptNotificationResponse) {
             final AcceptNotificationResponse acceptNotificationResponse = (AcceptNotificationResponse) message;
-            this.encoder().writeHeaders(context, acceptNotificationResponse.getStreamId(), SUCCESS_HEADERS, 0, true, writePromise);
+
+            final Http2Headers headers = new DefaultHttp2Headers()
```

Comment:
Come to think of it, this is probably a major contributor to performance issues. The success headers used to be completely static, and now we're allocating new headers for each notification. We might be able to get back to a reuse-the-headers pattern, but I'd prefer to defer that until we've demonstrated a clear need to the `apns-id` header at all.